Adobe Analytics Cloud Mode Update (#406)
rudderlabs/rudderstack-docs: destinations/analytics/adobe-analytics/setting-up-adobe-analytics-in-rudderstack.mdx

  </ul>
</div>

- **Send Both Timestamp and VisitorID for Timestamp Optional Reporting Suites**: This setting lets you choose whether to send both the timestamp and the `visitorID` when sending events to the server-side, in case you have [Timestamps Optional Reporting Suite](https://experienceleague.adobe.com/docs/analytics/admin/admin-tools/timestamp-optional.html?lang=en).

<div class="warningBlock">

> Enabling this setting might lead the data to be out of order. Hence, it is not recommended by Adobe. In order to make it work, you need to enable the <strong>Optional Timestamp</strong> setting in <strong>Reporting Suites</strong>.

Reviewer: In order to make it work, you need to enable the **Optional Timestamp** setting in **Reporting Suites**. 

1. Should it be Suite or Suites? 
2. How does one do that? I don't see this setting in the dashboard so assuming one has to go to the Reporting Suite dashboard?

</div>

> - **No Fallbacks for Visitor ID**: Enabling this setting removes the fallbacks. However, it is applicable only for the server-side when the **Drop Visitor ID** setting  is disabled and **marketingCloudId** is sent in the Adobe Analytics integration object. The priority order of setting the Adobe Analytics visitor ID is the destination-specific setting for `visitorId`, followed by `userId`, then `anonymousId`.

Reviewer: ```suggestion
- **No Fallbacks for Visitor ID**: Enabling this setting removes the fallbacks. However, it is applicable only for the server-side events when the **Drop Visitor ID** setting  is disabled and **marketingCloudId** is sent in the Adobe Analytics integration object. The priority order of setting the Adobe Analytics visitor ID is the destination-specific setting for `visitorId`, followed by `userId`, then `anonymousId`.
```

> - **No Fallbacks for Visitor ID**: Enabling this setting removes the fallbacks. However, it is applicable only for the server-side when the **Drop Visitor ID** setting  is disabled and **marketingCloudId** is sent in the Adobe Analytics integration object. The default behavior is to set the Adobe Analytics `visitorID` based on the destination-specific setting for `visitorId`, falling back to `userId` then `anonymousId`.

Reviewer: ```suggestion
- **No Fallbacks for Visitor ID**: Enabling this setting removes the fallbacks. However, it is applicable only for the server-side when the **Drop Visitor ID** setting  is disabled and **marketingCloudId** is sent in the Adobe Analytics integration object. The priority order of setting the Adobe Analytics visitor ID is the destination-specific setting for `visitorId`, followed by `userId`, then `anonymousId`.
```

Reviewer: Do we mean this? Currently, the meaning is not very clear.
